crates/primitives-traits/src/account.rs

```diff
@@ -128,6 +118,16 @@ impl Compact for Bytecode {
     }
 }
 
+impl From<&GenesisAccount> for Account {
```

Comment:
can we also get a `From<GenesisAccount>`

Reply:
well, yeah, but `From<&GenesisAccount>` is useful for all

Reply:
if you want, I can add this